### PR TITLE
Add docker hub auto build hook

### PR DESCRIPTION
**Pull request by andreymaznyak**
1. Add helm chart
1. Setup build and publish docker image on docker hub
Related issue #61

### CODE REVIEW

**Comment by andreymaznyak** [comment]
Thank you gues for this project. I deploy it to own kubernetes cluster. Pls add to documetation note about RESTRICTED_DOMAIN env variable. Pls add to documetation note about `RESTRICTED_DOMAIN` env variable. I was confused after start app with this variable.

**Comment by andreymaznyak** [comment]
@remiprev @simonprev Pls review

**Comment by simonprev** [comment]
Thank you for your interest in this project! I don't think we should merge in kubernetes _helm_ stuff since we don't use it internally and it is just config files. Would it possible to maintain a fork of this repo with the helm stuff? We could even mention it in the README if people want to use it 😄 

I'm interested about the docker part and publishing an official image on dockerhub. But we should tag it with "mirego" username. 

What do you think?

**Comment by andreymaznyak** [comment]
Yes, please setup docker auto build for this repo. For auto build you need build folder in this project. I change this merge request. And i move helm chart to other repo.

**Comment by andreymaznyak** [comment]
pls setup auto build for this repo, in your docker hub account, after it i change helm chart image for your image, Thank you :)
image

**Comment by andreymaznyak** [comment]
I added link to helm chart repo.

**Comment by andreymaznyak** [comment]
@simonprev Pls review again

File: hooks/build [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env sh
```
Comment (simonprev) [inline hooks/build]: This file is used by dockerhub?

**Comment by simonprev** [comment]
The changes have been added on `master` directly with e36915f04bbfe851c4b49b76c5487cd3cfe8a80c

Thank you for the helm chart example 👍

**Pull request closed without merging**